platform/admin/core/admin-core-insecuredefaults/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -27,9 +30,11 @@ xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0">
     </service>
 
     <bean id="usersPropertiesDeletionFlag"
```

Comment:
❗️ No longer using this bean. Which also means that the definition above of the collector isn't using the managed-properties.